### PR TITLE
Registered user checkout, Signout and Test ID updates.

### PR DESCRIPTION
**Pull request by devpatil7**

### CODE REVIEW

File: app/code/Magento/Pwa/Test/Mftf/Test/RegisteredUserCheckoutWithMultipleProducts.xml [hunk 1]
```diff
@@ -8,24 +8,26 @@
 
 <tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
        xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
-    <test name="GuestCheckoutWithMultipleProducts" extends="CreateConfigurableProduct">
+    <test name="RegisteredUserCheckoutWithMultipleProducts" extends="CreateConfigurableProduct">
```
Comment (tjwiebell) [inline app/code/Magento/Pwa/Test/Mftf/Test/RegisteredUserCheckoutWithMultipleProducts.xml:11]: Did you intend to repurpose this test, or would there be value in running the same test for both guests and customers?

File: app/code/Magento/Pwa/Test/Mftf/Test/RegisteredUserCheckoutWithMultipleProducts.xml [hunk 2]
```diff
@@ -8,24 +8,26 @@
...
-            <title value="Verify guest user should be able to checkout with Multiple products."/>
-            <description value="Verify guest user should be able to checkout with Multiple products."/>
+            <stories value="Checkout"/>
+            <title value="Verify registered user should be able to checkout with Multiple products."/>
+            <description value="Verify registered user should be able to checkout with Multiple products."/>
             <severity value="CRITICAL"/>
-            <testCaseId value="MC-15638"/>
+            <testCaseId value="MC-15720"/>
             <group value="PWA"/>
             <skip>
```
Comment (tjwiebell) [inline app/code/Magento/Pwa/Test/Mftf/Test/RegisteredUserCheckoutWithMultipleProducts.xml]: Should we unskip now and assume magento-research/pwa-studio#968 will be merged together?

File: app/code/Magento/Pwa/Test/Mftf/Test/RegisteredUserCheckoutWithMultipleProducts.xml (hunk 1 above)
Comment (devpatil7) [reply app/code/Magento/Pwa/Test/Mftf/Test/RegisteredUserCheckoutWithMultipleProducts.xml:11]: Yes. For guest checkout there are 2 tests so this one  I repurposed for registered user.

File: app/code/Magento/Pwa/Test/Mftf/Test/RegisteredUserCheckoutWithMultipleProducts.xml (hunk 2 above)
Comment (devpatil7) [reply app/code/Magento/Pwa/Test/Mftf/Test/RegisteredUserCheckoutWithMultipleProducts.xml]: I did unskip now as 1064 is closed. I will merge this with #968. Good to move it to QA column?

**Review by tjwiebell — CHANGES_REQUESTED** [review]
Consistently running into issue on Checkout step where shipping details are entered.

1cbc6b03ed292381

**Review by tjwiebell — APPROVED** [review]

**Comment by tjwiebell** [comment]
Holding `In QA` until @dpatil-magento can investigate failures related to tests not being run in isolation (data from previous test causes others to fail).

**Pull request merged**